tests/Functional/Fixtures/config/common_basic.yml

```diff
@@ -3,5 +3,8 @@ imports:
 
 services:
     logger:
-        class: Symfony\Component\Debug\BufferingLogger
+        class: Symfony\Component\ErrorHandler\BufferingLogger
         public: true
+        #class: Symfony\Component\HttpKernel\Log\Logger
```

Comment:
Could we remove commented code?

Reply:
yes I was debugging the tests, just forgot to remove it !